packages/@aws-cdk/aws-ec2/lib/vpc.ts

```diff
@@ -628,6 +634,17 @@ export interface VpcProps {
    */
   readonly natGatewaySubnets?: SubnetSelection;
 
+  /**
+   * What type of NAT provider to use
+   *
+   * Select between NAT gateways or NAT instances. NAT gateways
+   * may not be available in all AWS regions.
+   *
+   * @default - NatProvider.gateway()
```

Comment:
If you have a concrete value (`NatProvider.gateway()`) you should use it instead of `-`, no?

Reply:
Yes, you are right.